docs/OV_Runtime_UG/supported_plugins/config_properties.md

```diff
@@ -0,0 +1,226 @@
+# Query device properties, configuration {#openvino_docs_OV_UG_query_api}
```

Comment:
Maybe `Configuration of device properties`?

Reply:
But we also query read-only properties..
Don't know about proper name for the guide, suppose can be discussed once we have the whole picture of documents